### ISSUE TITLE
REQUEST: Alignment of colons too

### ISSUE THREAD

**Issue by erikvullings** [chemoish/atom-valign#4]
You've created a great plugin for Atom, and I use it a lot! 

Please, may I suggest a new feature. I was wondering if you could consider aligning the colons like you align the `=`. So instead of 
```
var obj = {
  "beer":       "belly",
  "pork belly": "yummy",
  "yummy":      42
};
```
align it like this
```
var obj = {
  "beer"      : "belly",
  "pork belly": "yummy",
  "yummy"     : 42
};
```
In case you prefer the current behavior, you could also add an extra option in the Settings so the user can decide which version he prefers.

**Comment by chemoish** [comment]
Hi @erikvullings,

I am considering refactoring this package to support configuration (possibly language configuration). This is the second time someone has requested some type of colon configuration.

I will consider this in the refactor.

**Comment by chemoish** [comment]
@erikvullings, lots of breaking changes with all the code modifications. Though the basic feature support for configuration is in there. It is a bit finicky because the implementation of the Settings View and how configuration trickles there...

Aside: Atom.config.get is overly cache heavy and I had a bunch of problems with it. Same with Settings View.

Please let me know if you run into any other bugs caused by the refactor.

Fixed by 6ed73e8
